index.js

```diff
@@ -91,6 +91,69 @@ say.speak = function(text, voice, speed, callback) {
   });
 };
 
+say.export = function(text, output, params, callback) {
```

Comment:
Can you make the method signature:

```javascript
function(text, voice, speed, filename, callback)
```

That way this function is called in a familiar method as `speak`. Using a params/options argument is a good idea, perhaps a pattern we'll switch to in the future.